### ISSUE TITLE
MAP_LOGO doesn't work as documented

### ISSUE THREAD

**Issue by seisman** [GenericMappingTools/gmt#3902]
**Description of the problem**

Here is what the documentation says:
```
MAP_LOGO
(-U) Specifies if a GMT logo with system timestamp should be plotted at the lower left corner of the plot [false].
```

As per the documentation, the following script should plot a GMT timestamp:
```
gmt set MAP_LOGO true
gmt psbasemap -R0/10/0/5 -JX10c/3c -Bx1 -By1 > map.ps
```
but it doesn't. It looks like a bug, but when we set MAP_LOGO to true, we definitely don't want to plot the timestamp for each plotting command. Maybe add the timestamp only for the first plot command (if GMT knows if the current command is the first one)? 

The following command works:
```
gmt psbasemap -R0/10/0/5 -JX10c/3c -Bx1 -By1 --MAP_LOGO=true > map.ps
```
but in this case, we can just use -U, which is much simpler.

**Comment by PaulWessel** [comment]
Yes, not sure what the intention was.  I think @remkos added this part, so perhaps he had some ideas for how this should work other than a plain substitute for **-U**.  I agree if all it does is replace **-U** with a longer option then it is not that helpful. Perhaps it was meant to be a mechanism to turn of -U statements in a script when you do not want them?  But looks like -U takes precedence.
